src/NUnitFramework/framework/Constraints/CollectionItemsEqualConstraint.cs

```diff
@@ -34,7 +34,10 @@ namespace NUnit.Framework.Constraints
     /// </summary>
     public abstract class CollectionItemsEqualConstraint : CollectionConstraint
     {
-        private readonly NUnitEqualityComparer comparer = NUnitEqualityComparer.Default;
+        /// <summary>
+        /// The NUnitEqualityComparer in use for this constraint
+        /// </summary>
+        private NUnitEqualityComparer _comparer = NUnitEqualityComparer.Default;
```

Comment:
Why are we removing the `readonly` modifier?

Reply:
It changed to a property and then back again - forgot to re-add it.